test/ModuleInterface/SmokeTest.swiftinterface

```diff
@@ -33,7 +32,7 @@ public class TestClass {
   // CHECK: public static var propWithNoAccessors: Int{{$}}
   public static var propWithNoAccessors: Int
 
-  // NEGATIVE-NOT: deinit
+  // CHECK: deinit
```

Comment:
Why did the printing behavior change here?

Reply:
Nevermind; I see elsewhere that the printing behavior was changed globally for deinit to accommodate the new modifier.